src/js/view/view.js

```diff
@@ -886,7 +886,8 @@ function View(_api, _model) {
             floatingPlayer = null;
 
             if (forever) {
-                _floatOnScroll = false;
+                _floatOnScroll = null;
+                _api.pause({ reason: 'interaction' });
```

Comment:
`stopFloating` can be called by the intersection observer, in which case the reason would not be interaction, correct ?
https://github.com/jwplayer/jwplayer/blob/7f093c7fabd2f08e0870543f158998b2172377a7/src/js/view/view.js#L880

Reply:
addressed